### PR TITLE
doc(:kr:): titles of decision-record-template-for-business-case

### PR DESCRIPTION
**Pull request by flavono123**
- 수준:
  - meaning of level in Korean
  - more natural expression when 'level' is used only itself
    - without an object??, such as the top-level `developer`
  - so,
    - 상위 -> 상위 수준
    - 하위 -> 하위 수준
- (최)
  - omit since it means the `most` but not necessary in this case
  - 최상위 -> 상위
- 상세 내용
  - 심층 다이빙 seems mis-translating from a machine translation
  - unnartural expression
  - translate 'deep-dive' as 'detail'

### CODE REVIEW

File: locales/ko/템플릿/비즈니스 사례에 대한 의사결정 기록 템플릿/index.md [hunk 1]
```diff
@@ -134,4 +134,3 @@
    * 요약
 
    * 세부 사항
-
```
Comment (flavono123) [inline locales/ko/템플릿/비즈니스 사례에 대한 의사결정 기록 템플릿/index.md:137]: this is another [removing trailing white spaces](https://github.com/joelparkerhenderson/architecture-decision-record/pull/67).
my local workspace has configured like this and it applied when i save the file.
- if i can remember, in future, i would check the diff just after saving a file gonna be edited.
- but i think this minor changes can be done on this time together :)
- or i can do that for the whole files at once if you want.

**Comment by joelparkerhenderson** [comment]
Thanks! I'll merge now. 

Next I'll look for options for dealing with trailing spaces and leading/trailing blank lines, such as with a git commit hook, or sed script, etc. Do you have a solution that you like?

**Comment by flavono123** [comment]
by my vscode settings since i use that editor:
```json
{
...
  // force EOF newline, trim trailing whitespace
  "files.insertFinalNewline": true,
  "files.trimTrailingWhitespace": true,
...
}
```

but we cannot enforce the "editor" for this works i think.
i guess there are more general solutions, such as git hook, as you said.

**Comment by flavono123** [comment]
https://git-scm.com/docs/git-stripspace
oh!

**Pull request merged**